### PR TITLE
added in pro pages that were deleted [ambassador-docs #193]

### PR DESCRIPTION
**Pull request by scoyle391**

### CODE REVIEW

**Comment by scoyle391** [comment]
@LukeShu I shifted some things around because I'm very nitpicky about table layout. Content wise, I updated some capitalization and added that one sentence at the top.

**Comment by nbkrause** [comment]
I assume the `reference/core/annotations` and `reference/pro/*` docs are already re-added in another PR?

**Comment by scoyle391** [comment]
> I assume the `reference/core/annotations` and `reference/pro/*` docs are already re-added in another PR?

@nbkrause annotations is already in there, and /pro pages got moved to be just in the /reference, but they exist as well

File: reference/config-format.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# The Ambassador Edge Stack Configuration Format
+
+The Ambassador Edge Stack exposes three methods for configuration.
+
+- [Ambassador Customer Resource Definitions (CRDs)](../../reference/core/crds)
```
Comment (LukeShu) [inline reference/config-format.md]: "custom", not "customer"

File: reference/upgrading.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Upgrading Ambassador Edge Stack
+
+Since Ambassador Edge Stack's configuration is entirely stored in annotations or a `ConfigMap`, no special process is necessary to upgrade Ambassador Edge Stack. If you're using the YAML files supplied by Datawire, you'll be able to upgrade simply by repeating the following `kubectl apply` commands.
```
Comment (LukeShu) [inline reference/upgrading.md]: replace "annotations or a `ConfigMap`" with "Kubernetes resources"

Comment (LukeShu) [inline reference/upgrading.md]: replace "annotations or a ConfigMap" with "Kubernetes resources"

File: doc-links.yml [hunk 3]
```diff
@@ -65,6 +67,10 @@
   items: 
     - title: Available Guides 
       link: /docs/guides/
+    - title: Operator Guide
+      link: /user-guide/operators
+    - title: Developer Guide
+      linkl: /user-guide/developers
```
Comment (LukeShu) [inline doc-links.yml]: typo: "link", not "linkl"

File: doc-links.yml [hunk 4]
```diff
@@ -96,6 +102,8 @@
       items:
       - title: Consul Integration
         link: /user-guide/consul
+      - title: Consul Connect Integration
+        link: /user-guide/consul-connect-ambassador
```
Comment (kelseyevans) [inline doc-links.yml]: have /user-guide/consul-connect-ambassador redirect to /user-guide/consul

File: reference/config-format.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# The Ambassador Edge Stack Configuration Format
+
+The Ambassador Edge Stack exposes three methods for configuration.
+
+- [Ambassador Custom Resource Definitions (CRDs)](../../reference/core/crds)
+- [Kubernetes Service Annotations](../../reference/core/annotations)
+- [Ambassador as an Ingress Controller](../../reference/core/ingress-controller)
+
+For most use cases, CRDs are the recommended configuration format since they directly expose all configuration options in a format that allows them to be managed as a separate object by Kubernetes. Because of this, all example configuration in the documentation is presented in CRD format. See [translating CRDs to annotations](../../reference/core/annotations#crd-translation) to translate CRD examples to Service annotations.
```
Comment (LukeShu) [inline reference/config-format.md]: These links to `/reference/core/annotations` might need updated?

File: doc-links.yml [hunk 6]
```diff
@@ -122,11 +128,19 @@
     - title: Configuring Ambassador Edge Stack
       link: /reference/configuration
     - title: Configuration Format
-      items:     
+      items:
+        - title: Format Overview
+          link: /reference/config-format
+        - title: Kubernetes Service Annotations  
+          link: /reference/core/annotations/
```
Comment (LukeShu) [inline doc-links.yml]: Um. You didn't re-add the `/reference/core/annotations` page.  Either add it, or remove this entry for it.

Comment (scoyle391) [reply doc-links.yml]: added

File: docs/dev-guide/service-preview.md [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+
+   ```yaml
+    ---
+    apiVersion: getambassador.io/v1
+    kind: Mapping
+    metadata:  
+      name: quote-ui
+    spec:
+      prefix: /
+      service: quote:5000
```
Comment (nbkrause) [inline docs/dev-guide/service-preview.md]: Remove

File: docs/dev-guide/service-preview.md [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: quote
+    spec:
+      ports:
+      - name: ui
+        port: 5000
+        targetPort: 5000
```
Comment (nbkrause) [inline docs/dev-guide/service-preview.md]: Remove

File: docs/dev-guide/service-preview.md [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+        metadata:
+          labels:
+            app: quote
+        spec:
+          containers:
+          - image: quay.io/datawire/tour:ui-$tourVersion$
+            name: quote-ui
+            ports:
+            - containerPort: 5000
+              name: http
```
Comment (nbkrause) [inline docs/dev-guide/service-preview.md]: Remove

File: docs/dev-guide/service-preview.md [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+          - image: quay.io/datawire/tour:backend-$tourVersion$
```
Comment (nbkrause) [inline docs/dev-guide/service-preview.md]: Change to quay.io/datawire/quote:0.2.7

File: docs/dev-guide/service-preview.md [hunk 11]
```diff
@@ -0,0 +1,169 @@
...
+<a class="apictl-dl" href="https://s3.amazonaws.com/datawire-static-files/apictl/$aproVersion$/darwin/amd64/apictl">Mac 64-bit</a> |
+<a class="apictl-linux-dl" href="https://s3.amazonaws.com/datawire-static-files/apictl/$aproVersion$/linux/amd64/apictl">Linux 64-bit</a>
+
+Make sure the client is somewhere on your PATH. In addition, place your license key in `~/.ambassador.key`.
+
+Information about open source code used in `apictl` can be found by running `apictl --version`.
+
+## Getting started
+
+In this quick start, we're going to preview a change we make to the backend service of the quote application, without impacting normal users of the application. Before getting started, make sure the [quote application is installed](../../../user-guide/install-ambassador-oss) on your cluster and you've installed the `apictl` command line tool, as explained above.
```
Comment (nbkrause) [inline docs/dev-guide/service-preview.md]:
```diff
- [quote application is installed](../../../user-guide/install-ambassador-oss)
+ [quote application is installed](../../../user-guide/install#create-a-mapping)
```

File: reference/authentication.md [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+* Modern infrastructure is fully distributed: cloud providers, on-premises data centers, laptops, mobile devices. Creating a strong perimeter across this diffuse edge is impractical.
+
+In the zero trust model, every request to a resource is verified, regardless of where that request originates. Google was one of the first companies to deploy a complete zero trust architecture, as detailed in their [BeyondCorp security architecture whitepaper](https://ai.google/research/pubs/pub43231).
+
+## Identity-Aware Proxy
+
+One of the key components of a zero trust architecture is the Identity-Aware Proxy. The Ambassador Edge Stack can be deployed in front of an application or microservices, and authenticate users, check authorization, and enforce other types of security policies. Critically, the Ambassador Edge Stack operates at the application level, which means it can take advantage of domain knowledge of users to improve security. Ambassador interfaces with the Identity Provider (IdP), which is the trusted canonical source for authentication and authorization information.
+
+![IAP](/doc-images/pro-iap.png)
```
Comment (LukeShu) [inline reference/authentication.md]:
```suggestion
![IAP](../../doc-images/pro-iap.png")
```

File: reference/core/annotations.md [hunk 13]
```diff
@@ -0,0 +1,184 @@
...
+For example:
+
+Using CRDs you can find all `Mapping`s deployed in the cluster with
+```
+kubectl get mappings -A
+```
+With annotations, you would have to describe and examine the `metadata.annotations` of every service.
+
+## Example
+The following is an example of how to configure annotations for the example quote service from the [install documentation](../../user-guide/getting-started).
```
Comment (LukeShu) [inline reference/core/annotations.md]:
```suggestion
The following is an example of how to configure annotations for the example quote service from the [install documentation](../../../user-guide/getting-started).
```

File: user-guide/operators.md [hunk 14]
```diff
@@ -0,0 +1,17 @@
...
+
+For more information, see the [Developer Guide](../../user-guide/developers).
+
+# Why Should Operators or Sysadmins Use Ambassador Edge Stack?
+
+Ambassador Edge Stack allows developers to manage individual service/API deployments, and frees time for operations to focus on global, cross-cutting concerns such as authentication, continuous delivery integration, rate limiting and tracing.
+
+In more detail, Ambassador Edge Stack supports operations in the following ways:
+
+* Is simple to [deploy and operate](../../user-guide/kubernetes-integration), relying entirely on Envoy and Kubernetes for routing and scaling
```
Comment (LukeShu) [inline user-guide/operators.md]: `user-guide/kubernetes-integration` doesn't exist.

File: user-guide/operators.md [hunk 15]
```diff
@@ -0,0 +1,17 @@
...
+* Has extensive support for [TLS termination](../../user-guide/tls-termination) and redirects
+* Integrated [diagnostics](/reference/statistics) and [tracing](../../user-guide/tracing-tutorial) for troubleshooting
```
Comment (LukeShu) [inline user-guide/operators.md]:
```suggestion
* Integrated [diagnostics](../../reference/statistics) and [tracing](../../user-guide/tracing-tutorial) for troubleshooting
```

File: docs/dev-guide/service-preview.md (hunk 11 above)
Comment (scoyle391) [reply docs/dev-guide/service-preview.md]: making this link to getting started, since we'll be switching those pages shortly.

**Pull request merged**